src/main/java/net/dv8tion/jda/core/entities/Message.java

```diff
@@ -131,6 +131,17 @@
     User getAuthor();
 
     /**
+     * Returns the author of this Message as a {@link net.dv8tion.jda.core.entities.Guild Guild} member.
+     * <br>This is just a shortcut to {@link #getGuild()} ()}{@link net.dv8tion.jda.core.entities.Guild#getMember(User)} () .getMember(getAuthor())}.
+     * <br><b>This is only valid if the Message was actually sent in a TextChannel.</b> This will return {@code null}
+     * if it was not sent from a TextChannel.
```

Comment:
I basically based it on the Message#getGuild() javadoc. Since I thought it had a familiar function.